src/app/components/AllowanceMenu/index.tsx

```diff
@@ -97,13 +100,13 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
         closeTimeoutMS={200}
         isOpen={modalIsOpen}
         onRequestClose={closeModal}
-        contentLabel="Allowance Options"
+        contentLabel={t("options_label")}
         overlayClassName="bg-black bg-opacity-25 fixed inset-0 flex justify-center items-center p-5"
         className="rounded-lg bg-white w-full max-w-lg"
       >
         <div className="p-5 flex justify-between dark:bg-surface-02dp">
           <h2 className="text-2xl font-bold dark:text-white">
-            Set a new budget
+            {t("set_budget")}
```

Comment:
Can this be `title`? Or title of a subsection?

Reply:
Title of subsection ig as this is the Allowance Menu component.